### ISSUE TITLE
Don't rely on git tag to determine the current version

### ISSUE THREAD

**Issue by moetayuko** [Morantron/tmux-fingers#125]
I use https://github.com/gpakosz/.tmux which patches TPM to perform shallow clones https://github.com/gpakosz/.tmux/blob/master/.tmux.conf#L1589-L1592, so the tags are not preserved. As a result, the CURRENT_GIT_VERSION can't be fetched, hence the install wizard is fired every time.

What about reading the current version from shard.yml, or moving the version string to a dedicated plain text file to avoid parsing yaml?

**Comment by Morantron** [comment]
I think we can read it from shard.yml without problems.

Can you try this commit?  https://github.com/Morantron/tmux-fingers/commit/5b1b380c994c94728a1ee53cf1073f381ff13f29

tpm allows pointing to specific refs like this

```
set -g @plugin "morantron/tmux-fingers#5b1b380c994c94728a1ee53cf1073f381ff13f29
```

**Comment by moetayuko** [comment]
Thanks, it works. BTW pinning sha1 doesn't work so I pinned the develop branch instead.

**Comment by Morantron** [comment]
Fixed in 2.2.1
